applications/crossbar/src/modules/cb_sup.erl

```diff
@@ -205,9 +205,9 @@ does_resource_exist(ModuleBin, FunctionBin, Args) ->
     try {module_name(ModuleBin), kz_term:to_atom(FunctionBin)} of
         {Module, Function} ->
```

Comment:
i think there's no need to call  `kz_module:is_exported` when `Module` is `false`
if taking changes suggested in kz_module, this `fun` would call `kz_module:is_exported(ModuleBin, Function, Arity)`